deps/rabbit/src/rabbit_networking.erl

```diff
@@ -819,3 +821,7 @@ ipv6_status(TestPort) ->
 ensure_listener_table_for_this_node() ->
     _ = ets:new(?ETS_TABLE, [named_table, public, bag, {keypos, #listener.node}]),
     ok.
+
+%% Needed for migration when feature flag listener_records_in_ets is enabled
+is_listener_for_this_node(#listener{node = Node}) ->
```

Comment:
How about `local_listener/1` or `listener_on_this_node/1`?

Reply:
took the name from the function above `ensure_listener_table_for_this_node()` but is_local_listener sounds much better.